.github/workflows/docker-image-release.yml

```diff
@@ -0,0 +1,27 @@
+name: docker-image-release
+
+on: workflow_dispatch
+
+jobs:
+  docker-image-build-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v4
+        with:
+          context: ./examples/docker/
+          file: ./examples/docker/Dockerfile
```

Comment:
I wonder if you can simply run `devbox generate dockerfile` command here, instead of pushing a hardcoded Dockerfile (so that it does not get out of sync)